### DISCUSSION TITLE
Would you consider also publishing this extension to Open VSX?

### DISCUSSION THREAD

**Discussion by neckspike** [edheltzel/nunjucks-for-visual-studio-code#17] [Ideas]
I prefer to use VSCodium which, like other non-MS derivatives of VS Code, can't use MS's extension repository. I've seen your plugin recommended a couple times and it looks really good tho! It seems pretty easy to publish to both repositories, I copied a bit from Open VSX's sample request with the relevant links here:

## What is Open VSX? Why does it exist?

Open VSX is a vendor neutral alternative to the MS marketplace used by most other derivatives of VS Code like [VSCodium](https://vscodium.com/), [Gitpod](https://www.gitpod.io), [OpenVSCode](https://github.com/gitpod-io/openvscode-server), [Theia](https://theia-ide.org/)-based IDEs, and so on.

You can read on about Open VSX at the Eclipse Foundation's [Open VSX FAQ](https://www.eclipse.org/legal/open-vsx-registry-faq/).

## How can you publish to Open VSX?

The docs to publish an extension can be found [here](https://github.com/eclipse/openvsx/wiki/Publishing-Extensions). This process is straightforward and shouldn't take too long. Essentially, you need an authentication token and to execute the `ovsx publish` command to publish your extension. There's also [a doc](https://github.com/open-vsx/publish-extensions/blob/master/docs/direct_publish_setup.md) explaining the whole process with an example GitHub Action workflow.

Thank you for reading!

**Comment by edheltzel** [comment]
@neckspike Yeah I'll check it out.

**Comment by neckspike** [comment]
Thank you : )

**Reply by edheltzel** [reply]
Submitted for the namespace - it seems like they are a couple of weeks behind on approvals: https://github.com/EclipseFdn/open-vsx.org/issues/2409

**Reply by neckspike** [reply]
Awesome, hopefully they'll work through their backlog soon

**Reply by edheltzel** [reply]
@neckspike https://open-vsx.org/extension/ginfuru/better-nunjucks

**Reply by neckspike** [reply]
Oh wow that was really fast!! Shows right up in VSCodium now and I already like it better than the other Nunjucks syntax plugin, the highlighting makes it much easier to read the code I'm trying to work with.

**Comment by cypressSap** [comment]
Could the newer releases please be added to Open VSX? Not sure if it's just an approval delay or something, but the newest version there is still 0.2.6. Thanks!
